controllers/util/storageconsumer.go

```diff
@@ -17,18 +17,19 @@ const (
 	TicketAnnotation              = "ocs.openshift.io/provider-onboarding-ticket"
 
 	// Constants for ConfigMap keys
-	rbdRadosNamespaceKey            = "rbd-rados-ns"
-	subVolumeGroupKey               = "cephfs-subvolumegroup"
-	subVolumeGroupRadosNamespaceKey = "cephfs-subvolumegroup-rados-ns"
-	csiRbdProvisionerCephUserKey    = "csi-rbd-provisioner-ceph-user"
-	csiRbdNodeCephUserKey           = "csi-rbd-node-ceph-user"
-	csiCephFsProvisionerCephUserKey = "csi-cephfs-provisioner-ceph-user"
-	csiCephFsNodeCephUserKey        = "csi-cephfs-node-ceph-user"
-	csiNfsProvisionerCephUserKey    = "csi-nfs-provisioner-ceph-user"
-	csiNfsNodeCephUserKey           = "csi-nfs-node-ceph-user"
-	rbdClientProfileKey             = "csiop-rbd-client-profile"
-	cephFsClientProfileKey          = "csiop-cephfs-client-profile"
-	nfsClientProfileKey             = "csiop-nfs-client-profile"
+	rbdRadosNamespaceKey                   = "rbd-rados-ns"
+	subVolumeGroupKey                      = "cephfs-subvolumegroup"
+	subVolumeGroupRadosNamespaceKey        = "cephfs-subvolumegroup-rados-ns"
+	csiRbdProvisionerCephUserKey           = "csi-rbd-provisioner-ceph-user"
+	csiRbdNodeCephUserKey                  = "csi-rbd-node-ceph-user"
+	csiCephFsProvisionerCephUserKey        = "csi-cephfs-provisioner-ceph-user"
+	csiCephFsNodeCephUserKey               = "csi-cephfs-node-ceph-user"
+	csiNfsProvisionerCephUserKey           = "csi-nfs-provisioner-ceph-user"
+	csiNfsNodeCephUserKey                  = "csi-nfs-node-ceph-user"
+	rbdClientProfileKey                    = "csiop-rbd-client-profile"
+	cephFsClientProfileKey                 = "csiop-cephfs-client-profile"
+	nfsClientProfileKey                    = "csiop-nfs-client-profile"
+	AnnotationNonResilientPoolsTopologyKey = "ocs.openshift.io/non-resilient-pools-topology-key"
```

Comment:
Pls move this to above stanza where we are exporting vars.

Current stanza is for keys in the configmap.